package.json

```diff
@@ -121,7 +125,9 @@
     "@types/use-sync-external-store": "0.0.6",
     "@vercel/analytics": "1.5.0",
     "@vitejs/plugin-react": "5.1.1",
+    "@vitest/browser": "4.0.12",
```

Comment:
This dependency can be removed unless it is required for this PR.

Reply:
I will remove.